### PR TITLE
fix(authn): Add OAuth  override commands

### PR DESCRIPTION
**Pull request by ttomsu**

### CODE REVIEW

**Review by lwander — APPROVED** [review]
Nice

File: setup/security/authentication/oauth/index.md [hunk 1]
```diff
@@ -40,10 +40,9 @@ hal config security oauth2 edit \
 
 ### Bring-Your-Own Provider
 
-TODO(ttomsu): Update this for Halyard config
-
 If you'd like to configure your own OAuth provider, you'll need to provide the following 
-configuration values in your `gate-local.yml` file:
+configuration values in your `gate-local.yml` file. If you're using Halyard, you can put this in 
+a new file at `~/.hal/default/staging/gate-local.yml`.
```
Comment (lwander) [inline setup/security/authentication/oauth/index.md]: Can you change `default` to `$DEPLOYMENT` and link to the custom profile setup?

Comment (ttomsu) [reply setup/security/authentication/oauth/index.md]: Done.

File: setup/security/authentication/oauth/index.md [hunk 2]
```diff
@@ -40,10 +40,10 @@ hal config security oauth2 edit \
 
 ### Bring-Your-Own Provider
 
-TODO(ttomsu): Update this for Halyard config
-
 If you'd like to configure your own OAuth provider, you'll need to provide the following 
-configuration values in your `gate-local.yml` file:
+configuration values in your `gate-local.yml` file. If you're using Halyard, you can put this in 
+a new file under your [deployment](/reference/halyard/#deployments) (typically `default`):
```
Comment (lwander) [inline setup/security/authentication/oauth/index.md:45]: Nice, I like how this reads

**Pull request merged**